### ISSUE TITLE
Can't set required translatable fields of Page model (title, slug, ...)

### ISSUE THREAD

**Issue by pyMan** [infoportugal/wagtail-modeltranslation#87]
Hello,
I can't find a way to make fields like 'title' and 'slug' required for languages other than the default one.

The next sample makes 'body' required for any language, but the 'required_languages' rule is ignored for the fields 'title' and 'slug'.

```
@register(TestPage)
class TestPageTR(TranslationOptions):
    fields = (
        'body',
    )
    required_languages = {'default': ('title', 'slug', 'body')}
```

A solution would be registering the Page model setting the 'required_languages' for those fields, but :
1. Page is already registered by wagtail modeltranslation (translation.py). So how unregister it?
2. This would affect any inherited model, not only the one I need.

**Comment by pyMan** [comment]
This won't work:

```
translator.unregister(Page)

from wagtail.wagtailcore.models import Page
from wagtail_modeltranslation.translation import TranslationOptions
from wagtail_modeltranslation.decorators import register
from wagtail_modeltranslation.translator import translator

translator.unregister(Page)

@register(Page)
class PageTR(TranslationOptions):
    fields = ('title', 'slug')
    required_languages = {'default': ('title', 'slug')}
```

It will raises some issues:

> django.core.management.base.SystemCheckError: SystemCheckError: System check identified some issues:
my_cms.TestPage.slug_en: (models.E006) The field 'slug_en' clashes with the field 'slug_en' from model 'wagtailcore.page'.
my_cms.TestPage.slug_it: (models.E006) The field 'slug_it' clashes with the field 'slug_it' from model 'wagtailcore.page'.
my_cms.TestPage.title_en: (models.E006) The field 'title_en' clashes with the field 'title_en' from model 'wagtailcore.page'.
my_cms.TestPage.title_it: (models.E006) The field 'title_it' clashes with the field 'title_it' from model 'wagtailcore.page'.

**Comment by pyMan** [comment]
I provided a fix to this problem with pull request #86

**Comment by Erwhann-Rouge** [comment]
This issue is still present in v0.6.0rc2.
